### PR TITLE
Managing OMERO docs re-organization

### PR DESCRIPTION
**Pull request by hflynn**
Partially based on the suggestions on https://docs.google.com/spreadsheets/d/1Swi9irN-GLqsAoYkRmpoGvoSoLGVDxfLh8UpwMcz1Mg/edit#gid=0 this highlights a managing OMERO subsection of the sysadmin docs aimed at restricted admins and better cross-references the CLI docs for this user group. 

This is a conservative approach with only updates one URL which is currently only 'live' in milestone releases anyway.

### CODE REVIEW

**Comment by hflynn** [comment]
I didn't move the CLI docs around because changing ownership can be done by group owners (more user-like) and I considered creating a manager-cli folder but actually there is proper sysadmin stuff on all the sysadmin/cli pages - light-admins.rst is rather scary and really requires a full admin's understanding of privileges and even the managing groups and users page also includes ldap content.

**Comment by hflynn** [comment]
Rebuilt on https://ci.openmicroscopy.org/view/Docs/job/OMERO-DEV-merge-docs/856/

**Comment by joshmoore** [comment]
Few comments:

 * I think this catches the most egregious red items from the spreadsheet.
 * If we foresee adding more to this section, having the anchor be `#managing-omero` rather than `#managing-omero-users-and-data` may be more friendly.
 * There's not yet much about managing data.
 * The provided subpages are quite technical quite quickly. I don't know how to solve this without starting to duplicate from http://help.openmicroscopy.org/facility-manager.html#manage
 * One point that it may make sense to get across for the "managing-omero" section is that in many cases this is a hand-off point, since the OS-level installation person may not have the necessary information.

**Comment by hflynn** [comment]
@joshmoore added some blurb and links to the facility manager guide and made the heading shorter.

File: omero/sysadmins/index.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@
 System Administrator Documentation
 ##################################
 
+This documentation begins with information aimed at OS-level administrators
+and moves on to day-to-day management of OMERO for facility
+administrators/managers (who may find it useful to read the :help:`Facility Managers help guide <facility-manager.html>` for an overview first).
```
Comment (pwalczysko) [inline omero/sysadmins/index.rst]: Never heard the word ``facility administrator`` before - it sounds like a forced transformation from ``system administrator`` (not sure though how to replace now)

File: omero/sysadmins/index.rst [hunk 2]
```diff
@@ -84,12 +88,12 @@ performance and security.
...
-Managing OMERO users and data
-*****************************
+**************
+Managing OMERO
+**************
 
-This section contains details on how user roles and permissions work in OMERO.
-New in OMERO 5.4.0, full administrators can now create restricted
+This section contains details on how to manage users, groups and data access
+in OMERO. New in OMERO 5.4.0, full administrators can now create restricted
```
Comment (pwalczysko) [inline omero/sysadmins/index.rst]: ``full administrators can now create...`` -> not precise, restricted administrators can create restricted administrators too (but I understand we need to keep it simple here)

Comment (hflynn) [reply omero/sysadmins/index.rst]: A full administrator needs to do it the first time though and that's too much detail for an intro sentence.

File: omero/sysadmins/index.rst (hunk 1 above)
Comment (hflynn) [reply omero/sysadmins/index.rst]: I guess I was thinking facility-level admin as opposed to operating system admin - I could add that clarification or remove the term completely if you prefer?

File: omero/sysadmins/index.rst (hunk 2 above)
Comment (pwalczysko) [reply omero/sysadmins/index.rst]: yes, probably too much detail and complication, let's leave it as it is

File: omero/sysadmins/index.rst (hunk 1 above)
Comment (pwalczysko) [reply omero/sysadmins/index.rst]: I think so, thank you

File: omero/users/whatsnew.rst [hunk 3]
```diff
@@ -4,7 +4,7 @@ What's new for OMERO 5.4 for users
 Updates and new features for OMERO 5.4 include:
 
 - Your system administrator can now create
-  :doc:`/sysadmins/admins-with-restricted-privileges` to allow facility
+  :doc:`/sysadmins/restricted-admins` to allow facility
```
Comment (pwalczysko) [inline omero/users/whatsnew.rst:7]: As discussed, add the link to the whatsnew for users from the users index page @hflynn

Comment (hflynn) [reply omero/users/whatsnew.rst:7]: Pushed now @pwalczysko is that location clear enough or do you want me to add it to a bullet list instead?

**Comment by pwalczysko** [comment]
Checked all the links. The new texts look good.  Maybe one more full build would be good as new commits came in.

File: omero/users/whatsnew.rst (hunk 3 above)
Comment (pwalczysko) [reply omero/users/whatsnew.rst:7]: Relisted for tomorrow with my name. Will have one more look at the commits pushed today

**Comment by joshmoore** [comment]
Comments from a general reading of this page:

 * I could imagine dropping `server-overview`.
 * "overview" in the installation guide titles is now a bit misleading.
 * `troubleshooting` might should be listed first in its section, though perhaps the maint/upgrading section might need further structure in general which can go along with the cli reworking later.
 * `OMERO_HOME` might should just be under troubleshooting.

Independent of the above, I think this is a definite step up.

**Comment by pwalczysko** [comment]
Repeated review today:
  - the new commits build and read well
  - the question of @hflynn : yes, this is a clear enough location

Ready to merge FMPOV -  but @joshmoore's comments will need some attention yet.

**Comment by hflynn** [comment]
@joshmoore I've left server overview as harmless, removed the 'overview' ending from the page titles for both the installation pages, moved troubleshooting and made the omero_home page just linked from a see also box at the top of troubleshooting.

As part of the review I noticed we still have the 5.2 on Windows to 5.3 on Linux page here which is inconsistent with the changes to the server upgrade page (which now refers people specifically to the 5.3.x docs if they are upgrading from 5.2) and also of course, points to the wrong pages now. Rather than update the links to point to 5.3.x and have people get lost swapping between doc versions, I'm just going to open a fresh PR to refer people directly to the 5.3 docs for all that content.

**Comment by hflynn** [comment]
Merge build updated

**Comment by hflynn** [comment]
@joshmoore happy for me to merge this now?

**Comment by joshmoore** [comment]
WFM.

**Pull request merged**